workspace/apps/pidp/src/app/features/organization-info/pages/endorsements/endorsements.page.ts

```diff
@@ -315,4 +316,25 @@ export class EndorsementsPage
       }),
     );
   }
+
+  public getStatus(endorsementRequestStatus: EndorsementRequestStatus): string {
+    let statusText = 'Requested';
+    if (endorsementRequestStatus === EndorsementRequestStatus.CANCELLED) {
+      statusText = 'Cancelled';
+    } else if (endorsementRequestStatus === EndorsementRequestStatus.DECLINED) {
+      statusText = 'Declined';
+    } else if (endorsementRequestStatus === EndorsementRequestStatus.APPROVED) {
+      statusText = 'In progress';
```

Comment:
For Approved why it's shown as In Progress?. This is the expected functionality ?

Reply:
Yes, at this point this is the second step in the endorsement process, there is one more step remaining for completion of the Endorsement.